### ISSUE TITLE
'All filters' button still shows when all the filters are visible

### ISSUE THREAD

**Issue by gestchild** [wellcomecollection/wellcomecollection.org#10391]
**What**
We have an 'All filters' button to get to filters when we don't have enough room to display them all on screen:

<img width="725" alt="Screenshot 2023-10-30 at 11 59 41" src="https://github.com/wellcomecollection/wellcomecollection.org/assets/6051896/be46c1bb-37e8-450d-8508-e5bc94c6a7c4">

However, the button is still visible when all the filters do fit on the screen:

<img width="1299" alt="Screenshot 2023-10-30 at 11 59 51" src="https://github.com/wellcomecollection/wellcomecollection.org/assets/6051896/caf03ece-19e3-4ba7-a5d2-9d258aeb53bf">

This can be confusing, as it implies there are more filters

**Where**
https://wellcomecollection.org/search/works?query=tree

**Comment by gestchild** [comment]
Thinking about this a bit more, I'm not entirely sure it is a bug or a feature, as it does allow access to all the filters at once without having to click on each individual dropdown:

<img width="708" alt="Screenshot 2023-10-30 at 12 07 23" src="https://github.com/wellcomecollection/wellcomecollection.org/assets/6051896/f18c7f16-1dc8-4613-bfc1-354c9143e213">

However, I do think it implies additional filters and found myself comparing the dropdown filters to the ones in the modal to discover that there weren't any. @j-jaworski what do you think?

**Comment by rcantin-w** [comment]
Just from my hazy memories in terms of what might have happened... From what I remember - it was intended as is.  I feel like I raised it but we did not have a UX designer when we actually built it to discuss the "live" version. Jonathan and Gareth O had talked about search at length before I joined and Jonathan was the decision maker after Gareth left, but I'm sorry that I can't fully recall the logic, I think you're right Gareth that it was about exposing them all in one place. [See mockup from Gareth, they show the All Filters even if there are no other filters to show](https://www.sketch.com/s/8ceb22c3-98d5-40a9-99e0-57731d69775b/prototype/998FC84B-03D0-4C28-8745-184210545C78/a/2F6E307E-0931-42A6-8DB2-2A3F273446B3).

I also believe we changed the label from More filters to All filters to reflect the change ([see before](https://web.archive.org/web/20211129170401/wellcomecollection.org/works)), so that also indicates intention.

Hope that helps a bit with extra context!
